core/include/cib.h

```diff
@@ -65,12 +65,24 @@ static inline void cib_init(cib_t *__restrict cib, unsigned int size)
  *                      Must not be NULL.
  * @return How often cib_get() can be called before the CIB is empty.
  */
-static inline unsigned int cib_avail(cib_t *__restrict cib)
+static inline unsigned int cib_avail(const cib_t *cib)
 {
     return cib->write_count - cib->read_count;
 }
 
 /**
+ * @brief Check if cib is full.
+ *
+ * @param[in] cib       the cib_t to check.
+ *                      Must not be NULL.
+ * @return      1 if cib_put() would return "-1", 0 otherwise
+ */
+static inline unsigned int cib_full(const cib_t *cib)
+{
+    return (int) cib_avail(cib) > (int) cib->mask;
```

Comment:
Clarity about the intent of the value. An `int` could mean anything, a `bool` can just be `true` or `false`.